src/main/webapp/app/shared/sidebar/sidebar.component.scss

```diff
@@ -41,3 +41,65 @@
         display: none !important;
     }
 }
+
+.dropdown-container,
+.btn-primary,
+.dropdown-menu,
+.dropdown-item {
+    box-sizing: border-box;
+}
+
+.dropdown-container {
+    position: relative;
+    width: 100%;
+    display: block;
+}
+
+.btn-primary {
+    width: 100%;
+}
+
+.btn {
+    font-size: 0.88em;
+}
+
+.dropdown-menu {
+    position: absolute;
+    top: 100%;
+    left: 0;
+    right: 0;
+    background-color: white;
+    border: 1px solid #ccc;
```

Comment:
You should use the color variables bg-secondary, etc. (or similar) to choose the right colors also when switching to darkmode. You can get the correct variable through the Chrome Dev tools by selection text for example or the background of e.g. the Sidebar itself. (I think the variables are at the bottom of the styles bar in the Dev tools)